common/search/src/main/scala/weco/api/search/config/builders/MetricsBuilder.scala

```diff
@@ -0,0 +1,31 @@
+package weco.api.search.config.builders
+
+import weco.monitoring.Metrics
+import weco.monitoring.memory.MemoryMetrics
+import weco.monitoring.typesafe.CloudWatchBuilder
+
+import akka.stream.Materializer
+import com.typesafe.config.Config
+
+import scala.concurrent.Future
+import scala.util.Try
+import scala.concurrent.ExecutionContext
+
+/**
+  * Build an appropriate Metrics object for use by weco.http.monitoring.HttpMetrics
+  * For a "Real" instance, publish metrics on CloudWatch,
+  * For a local or experimental instance, simply store them in memory.
+  */
+object MetricsBuilder {
+  def apply(config: Config)(
+    implicit
+    materializer: Materializer,
+    ec: ExecutionContext
+  ): Metrics[Future] =
+    sys.env.get("API_USE_MEMORY_METRICS") match {
```

Comment:
As with API_SETTINGS_MODE, I think we'd be better off changing this to a MODE setting rather than a boolean.

In particular, I can imagine it might be useful to change this to a third implementation of Metrics that logs to stdout (I think MemoryMetrics holds the values but doesn't send them anywhere).